src/lib/components/Swap/Input.tsx

```diff
@@ -8,6 +8,7 @@ import {
   useSwapCurrencyAmount,
   useSwapInfo,
 } from 'lib/hooks/swap'
+import useInputOutputFieldAmount from 'lib/hooks/swap/useInputOutputFieldAmount'
```

Comment:
As a hook specific to Input/Output, I think this should be inlined to Input.tsx and not in a separate file. I don't think it's weird to export from Input and import in Output.